nucypher/blockchain/eth/agents.py

```diff
@@ -298,7 +278,7 @@ def partition_stakers_by_activity(self) -> Tuple[List[str], List[str], List[str]
         return active_stakers, pending_stakers, missing_stakers
 
     @contract_api(CONTRACT_CALL)
-    def get_all_active_stakers(self, periods: int, pagination_size: int = None) -> Tuple[int, List[str]]:
+    def get_all_active_stakers(self, periods: int, pagination_size: Optional[int] = None) -> Tuple[NuNits, List[List[int]]]:
```

Comment:
Contract returns value in some not very intuitive way: address of staker put inside uint256 type, so in line 307 we're doing coversion fron int to address. So returning type here should be something like `Tuple[NuNits, List[Tuple[ChecksumAddress, NuNits]]]`

Reply:
:heavy_check_mark: - with new TODO to follow-up.

Reply:
Rolled back this change (a little).  Let's get another check from @vzotova.